packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -268,7 +268,7 @@ export class Popover
 
   @State() defaultMessages: PopoverMessages;
 
-  arrowEl: SVGElement;
+  arrowEl: SVGSVGElement;
```

Comment:
the correct type for the `<svg>` element is `SVGSVGElement`.
`SVGElement` means "any element from the SVG namespace", just like `HTMLElement` means "any element from the HTML namespace"